### DISCUSSION TITLE
webstorm auto fix wxml

### DISCUSSION THREAD

**Discussion by garrettzxd** [wxmlfile/eslint-plugin-wxml#62] [Q&A] [answered]
## webstorm下wxml可以检测到异常，但是不能自动修复

![image](https://user-images.githubusercontent.com/13514116/194828021-5a130084-1cb1-4011-ae5e-30dbe192cf4f.png)

## 以下命令不能生效
```shell
eslint miniprogram/pages/index/index.wxml --fix
```
webstorm要怎么配置？

**Comment by iChenLei** [comment, marked as answer]
这个是 prettier 的功能？ (eslint-plugin-wxml不支持autofix
